app/src/main/java/com/sun_asterisk/moviedb_50/screen/home/HomePresenter.kt

```diff
@@ -1,42 +1,52 @@
 package com.sun_asterisk.moviedb_50.screen.home
 
 import com.sun_asterisk.moviedb_50.data.model.Genres
+import com.sun_asterisk.moviedb_50.data.model.GenresResponse
 import com.sun_asterisk.moviedb_50.data.model.Movie
+import com.sun_asterisk.moviedb_50.data.model.MoviesResponse
 import com.sun_asterisk.moviedb_50.data.source.remote.OnFetchDataJsonListener
 import com.sun_asterisk.moviedb_50.data.source.remote.Repository
+import com.sun_asterisk.moviedb_50.utils.Constant
 
 class HomePresenter(private val repository: Repository) : HomeContract.Presenter {
     private var view: HomeContract.View? = null
 
-    override fun onStart() {}
+    override fun onStart() {
+        getGenres()
+        getMovie(Constant.BASE_NOW_PLAYING, 1)
+    }
+
     override fun onStop() {}
+
     override fun setView(view: HomeContract.View?) {
         this.view = view
     }
 
     override
     fun getGenres() {
-        repository.getGenres(object : OnFetchDataJsonListener<MutableList<Genres>> {
+        repository.getGenres(object : OnFetchDataJsonListener<GenresResponse> {
 
             override fun onError(e: Exception) {
                 view?.onError(e)
             }
 
-            override fun onSuccess(data: MutableList<Genres>?) {
-                view?.onGetGenresSuccess(data)
+            override fun onSuccess(data: GenresResponse?) {
+                val genres = data?.let { it -> it.list.map { Genres(it) } }
+                view?.onGetGenresSuccess(genres)
```

Comment:
Viết ntn có gọn hơn k nhỉ (tùy quan điểm)
```suggestion
                data?: return
                view?.onGetGenresSuccess(data.list.map { Genres(it) })
```